paperqa/agents/task.py

```diff
@@ -487,3 +497,214 @@ def __len__(self) -> int:
     LitQAv2TaskDataset.__module__,
     LitQAv2TaskDataset.__name__,
 )
+
+
+class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment):
+    """Environment to evaluate paperqa's vs human's answers on Long Form RAG QA questions."""
+
+    def __init__(
+        self,
+        *args,
+        qid: str | UUID,
+        question: str,
+        human_answer: str,
+        gt_doc_ids: list[str],
```

Comment:
Is it possible to use `str | UUID` here? Or does LFRQA not match `UUID`

If it's `int` convertible let's use `int`